client/src/pages/PostForm/AskForm/AskForm.component.tsx

```diff
@@ -105,128 +117,122 @@ const AskForm = ({
 
   return (
     <form onSubmit={internalOnSubmit}>
-      <div className="question-form">
-        <div className="question-layout">
-          <div className="grid title-grid">
-            <label className="form-label">
-              Question
-              <p>Give your question a short and relevant title</p>
-            </label>
-            <input
-              {...register('postTitle', {
-                minLength: 15,
-                maxLength: TITLE_MAX_LEN,
-                required: true,
-              })}
+      <FormControl>
+        <FormLabel sx={styles.formLabel}>Question</FormLabel>
+        <FormHelperText sx={styles.formHelperText}>
+          Give your question a short and relevant title
+        </FormHelperText>
+        <Input
+          placeholder="Field Empty"
+          {...register('postTitle', {
+            minLength: 15,
+            maxLength: TITLE_MAX_LEN,
+            required: true,
+          })}
+        />
+        {formErrors.postTitle ? (
+          <Alert status="error" mt="4px">
+            <AlertIcon />
+            Please enter a title with 15-150 characters.
+          </Alert>
+        ) : (
+          <Box sx={styles.charsRemainingBox}>
+            {titleCharsRemaining} characters left
+          </Box>
+        )}
+      </FormControl>
+      <FormControl sx={styles.formControl}>
+        <HStack>
+          <Box sx={styles.formLabel}>Description</Box>
+          <Box textStyle="body-2">(optional)</Box>
+        </HStack>
+
+        <FormHelperText sx={styles.formHelperText}>
+          Additional description to give more context to your question
+        </FormHelperText>
+        <Controller
+          name="postDescription"
+          control={control}
+          rules={{
+            validate: (v) => !replaceEmptyRichTextInput(v) || v.length >= 30,
+          }}
+          render={({ field: { onChange, value, ref } }) => (
+            <RichTextEditor
+              onChange={onChange}
+              value={value}
+              editorRef={ref}
+              wrapperStyle={{ height: '148px' }}
+              editorStyle={{ height: `${148 - (26 + 6 * 2)}px` }}
```

Comment:
Consider extracting this into the Chakra theme, then do something like...
```suggestion
              {...styles.richTextEditor}
```